src/CommunityToolkit.Maui/Views/Expander/Expander.shared.cs

```diff
@@ -251,6 +251,12 @@ void ResizeExpanderInItemsView(TappedEventArgs tappedEventArgs)
 				ForceUpdateCellSize(collectionView, size, tapLocation);
 			}
 #endif
+#if IOS || MACCATALYST
+            else if (element is ScrollView scrollView)
+			{
+			    (scrollView as IView).InvalidateMeasure();
```

Comment:
```suggestion
			    ((IView)scrollView).InvalidateMeasure();
```

Reply:
@VladislavAntonyuk , Thanks for the correction.